packages/vite/src/node/config.ts

```diff
@@ -67,7 +69,7 @@ export interface UserConfig {
   /**
    * Array of vite plugins to use.
    */
-  plugins?: (Plugin | Plugin[])[]
+  plugins?: (PluginOption | PluginOption[])[]
```

Comment:
**question:** In which situation is an array with possible _falsy_ values helpful?

Reply:
@matias-capeletto Is it wanted behavior that plugins can hold an array of **array** of plugins? Looks weird, but I currently don't know better.

Reply:
Yes, it is used to create presets, a plugin can return several plugins. You can check out the vue plugin for an example